src/PKSim.Core/Commands/RenameMoleculeInSimulationSubjectCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Extensions;
+using PKSim.Core.Extensions;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Commands
+{
+   public class RenameMoleculeInSimulationSubjectCommand : RenameEntityCommand
```

Comment:
Rename a molecule becomes much more complicated. You have to rename formulas, sub containers etc..
Test coming